grafana-plugin/src/components/MSTeams/MSTeamsInstructions.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { FC } from 'react';
+
+import { Button, Icon, VerticalGroup, Field, Input } from '@grafana/ui';
+import cn from 'classnames/bind';
+import CopyToClipboard from 'react-copy-to-clipboard';
+
+import Block from 'components/GBlock/Block';
+import PluginLink from 'components/PluginLink/PluginLink';
+import Text from 'components/Text/Text';
+import MSTeamsLogo from 'icons/MSTeamsLogo';
+import { useStore } from 'state/useStore';
+import { openNotification, openWarningNotification } from 'utils';
+
+import styles from './MSTeamsInstructions.module.css';
+
+interface MSTeamsInstructionsProps {
+  onCallisAdded?: boolean;
+  showInfoBox?: boolean;
+  personalSettings?: boolean;
+  verificationCode: string;
+  onHide?: () => void;
+}
+
+const cx = cn.bind(styles);
+
+const MSTeamsInstructions: FC<MSTeamsInstructionsProps> = (props) => {
+  const { onCallisAdded, showInfoBox, personalSettings, onHide = () => {}, verificationCode } = props;
+  const store = useStore();
+  const { msteamsChannelStore } = store;
```

Comment:
We can destructure in one line:
```
const { msteamsChannelStore } = useStore();
```

Also this component should be wrapped with `observer`